doc/changes/changes_5.0.0.md

```diff
@@ -6,6 +6,10 @@ Code name: Removed `SQL_DIALECT` property
 
 The `SQL_DIALECT` property is obsolete. Please, do not provide this property anymore.
 
+## Feature
+
+* #32: Get Geometry srid from a system table instead of the hardcoded value.
```

Comment:
```suggestion
* #32: Get Geometry SRID from a system table instead of the hardcoded value.
```
Add a link to a Wikipedia article about SRIDs behind the name. Few users will know what that is.